documentation/docs/plugins/portal-examples.md

```diff
@@ -0,0 +1,45 @@
+---
+displayed_sidebar: pluginsSidebar
+---
+
+# Portal's Plugins Examples
+
+### Adding a tab in the PBX dashboard page
+
+```js
+(async () => {
+  await app.initialize();
+  const context = app.getContext();
+
+  document.getElementById('name').innerHTML = context.app.extra.record.auth.username;
+})();
```

Comment:
Does it actually need to be in an anonymous function?

Reply:
NoYes, it depends how we see it. Inside an HTML document it valid and the shorter form. But inside a framework it's not useful.

🤔 Not sure which way we want to expose at-first. It's a good point to discuss